util/config/generate-copyright

```diff
@@ -2,13 +2,19 @@
 
 import os
 import argparse
+import subprocess
 
 parser = argparse.ArgumentParser(description='Write the COPYRIGHT file suitable for importing')
 parser.add_argument('source', help='The source directory where COPYRIGHT is found')
 parser.add_argument('destination', help='The location to write the updated COPYRIGHT file')
+parser.add_argument('--chpl-home', dest='chpl_home', help='the location of the chapel source')
 
 args = parser.parse_args()
 
+out_file = os.path.join(args.destination, 'COPYRIGHT')
+tmp_file = os.path.join(args.destination, 'COPYRIGHT.in')
```

Comment:
Does `update_if_different` handle removing `COPYRIGHT.in` if the files are not the same? Just want to make sure we don't have a temporary file hanging around. Could we not just open a temporary file for this?